src/components/expired-subscription-modal/index.jsx

```diff
@@ -9,20 +10,6 @@ const ExpiredSubscriptionModal = () => {
   if (!customerAgreement?.hasCustomLicenseExpirationMessaging) {
     return null;
   }
-  const onClickHandler = () => {
-    let url = customerAgreement?.urlForButtonInModal;
-
-    if (url) {
-      // Check if the URL starts with 'http://' or 'https://'
-      if (!url.startsWith('http://') && !url.startsWith('https://')) {
-        // Prepend 'https://' if the URL is missing the protocol
-        url = `https://${url}`;
-      }
-
-      // Navigate to the full URL
-      window.open(url, '_blank'); // Opening in a new tab
-    }
-  };
   return (
     <AlertModal
       title={<h3 className="mb-2">{customerAgreement?.modalHeaderText}</h3>}
```

Comment:
nit: given the early return above with `if (!customerAgreement?.hasCustomLicenseExpirationMessaging) {`, I believe you can assume `customerAgreement` exists here and below, where the `?.` is no longer needed.